source/libs/icons.js

```diff
@@ -40,8 +40,8 @@ export const branch = () => <svg aria-hidden="true" class="octicon octicon-git-b
 
 export const privateLock = () => <svg className="octicon octicon-lock" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 13H3v-1h1v1zm8-6v7c0 .55-.45 1-1 1H1c-.55 0-1-.45-1-1V7c0-.55.45-1 1-1h1V4c0-2.2 1.8-4 4-4s4 1.8 4 4v2h1c.55 0 1 .45 1 1zM3.8 6h4.41V4c0-1.22-.98-2.2-2.2-2.2-1.22 0-2.2.98-2.2 2.2v2H3.8zM11 7H2v7h9V7zM4 8H3v1h1V8zm0 2H3v1h1v-1z" /></svg>;
 
-export const privateLockFilled = () =>
-	<svg version="1.1" className="octicon octicon-lock" height="16.426" width="13.426" viewBox="-118 118 13.426 16.426">
+export const privateLockFilled = (width, height) =>
+	<svg version="1.1" className="octicon octicon-lock" height={height} width={width} viewBox={`-118 118 ${width} ${height}`}>
```

Comment:
This is not correct. `viewBox` should not be touched.

Reply:
And it should be a `size` argument with one value that is used for both width/height. The viewBox should ensure to keep the aspect ratio.